tests/FakeItEasy.IntegrationTests.VB/RaisingEventsTests.vb

```diff
@@ -11,7 +11,7 @@ Public Interface IHaveEvents
     Event NonGenericEventHander As EventHandler
     Event GenericEventHander As EventHandler(Of MyEventArgs)
 
-    <SuppressMessage("Microsoft.Design", "CA1009:DeclareEventHandlersCorrectly",
+    <SuppressMessage("Microsoft.Design", "CA1003:DeclareEventHandlersCorrectly",

```

Comment:
The code seems to have changed?

Reply:
Looks like there are two different warnings:
[CA1003](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1003): Use generic event handler instances
[CA1009](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1009): Declare event handlers correctly

I assume the analyzers changed to emit the new one in this case.

Reply:
You're right. I should update the um... title? Description?

Reply:
Probably, yes. I assume this is some kind of identifier, but I don't know where it comes from.
*or* you could just remove the `[SuppressMessage]` attribute and use the IDE to generate the `#pragma warning disable` for you.